assets/app/views/site/edit.js

```diff
@@ -3,7 +3,7 @@ var fs = require('fs');
 var Backbone = require('backbone');
 var _ = require('underscore');
 
-var templateHtml = fs.readFileSync(__dirname + '/../templates/SiteEditTemplate.html').toString();
+var templateHtml = fs.readFileSync(__dirname + '/../../templates/site/edit.html').toString();
```

Comment:
Right, this `readFileSync` approach is probably the most common way to browserify templates. You could use [stringify](https://github.com/JohnPostlethwait/stringify) to `require` html / text files (we use it on Midas). But it's just an esthetic difference.